lib/components/molecules/Accordion/AccordionPanel/AccordionPanel.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+/**
+ *
+ * AccordionPanel
+ *
+ */
+import React from 'react';
+import type { Node } from 'react';
+import styled from 'styled-components';
+
+import styles from './AccordionPanel.style';
+
+type Props = {
+  className?: string,
+  children: Node,
+};
+
+const AccordionPanel = ({ children, className }: Props): Node => (
```

Comment:
This could have been just a local component to Accordion component itself instead of making it a full fledge component since it can't be used independently